doc/content/end-user-documentation/testing.rst

```diff
@@ -0,0 +1,176 @@
+
+
+Testing
+=======
+
+For testing Lona comes with a `pytest <http://docs.pytest.org>`_ plugin, based
+on pytest-aiohttp. The plugin contains two fixtures:
+`lona_app_context <#lona-app-context-example>`_ and
+`lona_project_context <#lona-project-context-example>`_.
+
+`lona_app_context <#lona-app-context-example>`_ spins up a test server using an
+API like the
+`Lona script interface </end-user-documentation/getting-started.html#running-lona-from-a-script>`_,
+`lona_project_context <#lona-project-context-example>`_ takes the project root
+and settings of a
+`Lona project </end-user-documentation/getting-started.html#starting-a-lona-project-from-the-project-template>`_
+to start the test server. Lona test servers run on an unprivileged random port.
+
```

Comment:
In the following examples you use ``playwright`` as browser-interface. But you never introduce it here.  Maybe add it to the list of tools like ``pytest`` and ``pytest-aiohttp``.
Maybe add some words about ``playwright`` vs other browser-testing frameworks here?

Reply:
Good point! I added a section about browser testing and a synchronous code example for frameworks like selenium

Reply:
Thanks! Looks good.